editor/filesystem_dock.cpp

```diff
@@ -1604,7 +1609,11 @@ void FileSystemDock::_update_dependencies_after_move(const HashMap<String, Strin
 		const Error err = ResourceLoader::rename_dependencies(file, p_renames);
 		if (err == OK) {
 			if (ResourceLoader::get_resource_type(file) == "PackedScene") {
-				scenes_to_reload.push_back(file);
+				if (file.hash64() == edited_scene_path_hash) {
```

Comment:
Not sure if it's worth it to use hash here. Usually we just compare the strings directly. Even with 100 opened scenes there won't be visible difference. (also not even sure if hashing every path is faster)

Reply:
Hashing here shouldn't be any faster if it's done on each path, hashing a string should take `O(n)` time, where `n` is the length of `file` string *always*, but comparing strings takes `O(n)` only when the strings match length, and exists early if they don't match, so hashing should be slower